charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -23,7 +23,8 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    # TODO: remove TEMPORARY OVERRIDE FOR GlobalSearch R2, reinstate: DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1130.service.core-compute-preview.internal/
```

Comment:
Temporary values should be updated prior to merge into `Global_Search_Release2` branch and then removed before that branch is merged for release.